File: src/main/java/com/epri/dss/shared/impl/PolarImpl.java
package com.epri.dss.shared.impl;

import com.epri.dss.shared.Polar;

public class PolarImpl implements Polar {

	private double mag;
	private double ang;

	public PolarImpl(double mag, double ang) {
		this.mag = mag;
		this.ang = ang;
	}

	public double getMag() {
		return mag;
	}
	public void setMag(double mag) {
		this.mag = mag;
	}

	public double getAng() {
		return ang;
	}
	public void setAng(double ang) {
		this.ang = ang;
	}

}

File: pydss/shared/PolarImpl.py
from dss.shared.Polar import (Polar,)


class PolarImpl(object, Polar):
    mag = None
    ang = None

    def __init__(self, mag, ang):
        self.mag = mag
        self.ang = ang

    def getMag(self):
        return self.mag

    def setMag(self, mag):
        self.mag = mag

    def getAng(self):
        return self.ang

    def setAng(self, ang):
        self.ang = ang
